Cargo.toml

```diff
@@ -81,10 +83,10 @@ winter-utils = { version = "0.12", default-features = false }
 [dev-dependencies]
 assert_matches = { version = "1.5", default-features = false }
 criterion = { version = "0.5", features = ["html_reports"] }
-getrandom = { version = "0.2", features = ["js"] }
+getrandom = { version = "0.3", default-features = false }
```

Comment:
Does this change not also require `wasm_js`? As per [these docs](https://docs.rs/getrandom/latest/getrandom/#webassembly-support)

Reply:
This is a dev-dependency, so I guess not, because we don't run tests in no-std, right?

Reply:
Ah, didn't catch that it was a dev dependency.

Reply:
Yes, as Philipp said, it is because there is no need to add it 👌🏼